### PR TITLE
Master

### PR DESCRIPTION
**Pull request by daodavid**
* Logistic Regression
  1) Cross entropy
  2) The origin of sigmoid
  3) Gradient descent Math
  4) Init Binary logistic regression from scratch implementation
* Math
  1) Maximum likelihood
  2) Logg Odds and Odds ratio investigation  
  3) Sigmoid explorations

### CODE REVIEW

**Comment by daodavid** [comment]
:+1:  +1: :+1:
